tamr_unify_client/models/project/categorization.py

```diff
@@ -15,4 +16,30 @@ def model(self):
         alias = self.api_path + "/categorizations/model"
         return MachineLearningModel(self.client, None, alias)
 
+    def create_taxonomy(self, name=""):
+        """Creates a Taxonomy for this Categorization project.
+
+        A taxonomy cannot already be associated with this project.
+
+        :returns: The new Taxonomy
+        :rtype: :class:`~tamr_unify_client.models.taxonomy.resource.Taxonomy
+        """
+        alias = self.api_path + "/taxonomy"
+        resource_json = self.client.post(
+            alias, json={"name":  name}
+        ).successful().json()
+        return Taxonomy.from_json(self.client, resource_json, alias)
+
+    def taxonomy(self):
+        """Retrieves the Taxonomy associated with Categorization project.
+
+        If a taxonomy is not already associated with this project, call create_taxonomy() first.
```

Comment:
what happens if you call taxonomy on a project that doesn't have one associated with it? should it raise an error and say something along the lines of "you need to create a taxonomy for this project" ?

Reply:
There's a 404 error